pkg/detectors/slackwebhook/slackwebhook.go

```diff
@@ -69,11 +70,10 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 				if err != nil {
 					continue
 				}
-				body := string(bodyBytes)
 
 				defer res.Body.Close()
 
-				if res.StatusCode >= 200 && res.StatusCode < 300 || (res.StatusCode == 400 && (strings.Contains(body, "no_text") || strings.Contains(body, "missing_text"))) {
+				if res.StatusCode >= 200 && res.StatusCode < 300 || (res.StatusCode == 400 && (bytes.Equal(bodyBytes, []byte("no_text")) || bytes.Equal(bodyBytes, []byte("missing_text")))) {
```

Comment:
What situations do these body checks actually handle?

Reply:
Depending on the version of slackwebhooks that were setup we get 400s with either "no_text" or "missing_text". These usually indicate the webhook is active but does not support empty text in the payload. At least that is my current understanding.